packages/flutter/lib/src/material/app.dart

```diff
@@ -1173,3 +1166,115 @@ class _MaterialAppState extends State<MaterialApp> {
     );
   }
 }
+
+enum _MaterialInspectorButtonVariant { filled, toggle, iconOnly }
```

Comment:
why do we not have an equivalent pattern for the cupertino button?

Reply:
Because I hadn't added it yet 🙃 It's added now

Reply:
Add dart doc 👍